lib/recurly.js

```diff
@@ -112,8 +109,7 @@ export class Recurly extends Emitter {
     this.reporter = new Reporter({ recurly: this });
     this.request = new Request({ recurly: this });
 
-    // @deprecated -- Old method of instantiating single-subscription Pricing
-    this.Pricing = () => new SubscriptionPricing(this);
+    this.Pricing = () => new SubscriptionPricing(this); // deprecated
```

Comment:
This is a deprecation note that only applies to this line -- wasn't clear before.